tests/contract-playground/src/auditor_mode/delegate_call_no_checks/DelegateCallWithoutAddressChecks.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8;
+
+contract DelegatecallWithoutAddressChecks {
+
+    // Any delegate calls to addresses in state variables, we don't check!
+    // Because we assume that other detectors catch setting of variables 
+    // either in the constructor or in other methods with modifiers.
+    address public manager;
+
+    constructor() {
+    
+    }
+
+    function delegate1(address to, bytes memory data) external {
+        to.delegatecall(data); // `to` is not protected, therefore BAD
+    }
```

Comment:
Here. we can see that this function doesn't perform any checks yet, makes a delegate call on an address that is not a state variable.

So it's considered BAD (and we capture it)